src/de/impl_core.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(unused_unsafe)]
+#![allow(clippy::needless_borrow)]
```

Comment:
is this still needed after your clippy fix?

Reply:
Nope, removed!